sample/src/main/java/kin/core/sample/OnBoarding.java

```diff
@@ -19,10 +19,10 @@
 
 class OnBoarding {
 
-    private static final String URL_CREATE_ACCOUNT = "http://friendbot-playground.kininfrastructure.com/?addr=";
+    private static final String URL_CREATE_ACCOUNT = "https://friendbot-playground.kininfrastructure.com/?addr=";
     private static final int FUND_KIN_AMOUNT = 6000;
     private static final String URL_FUND =
-        "http://faucet-playground.kininfrastructure.com/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+        "https://faucet-playground.kininfrastructure.com/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
```

Comment:
playground faucet has no https, so we can't change it.
because it's a sample app, I suggest to simply allow http in android 9 (something to add in the manifest)

Reply:
ok